mobile/src/main/res/values-de/strings.xml

```diff
@@ -1,135 +1,143 @@
+  <string name="info_write_tag_finished">Erfolgreich abgeschlossen</string>
+  <string name="info_write_failed">Fehler beim Schreiben des NFC-Tags</string>
+  <string name="info_write_tag_unsupported">Dieses Gerät unterstützt kein NFC</string>
+  <string name="info_write_tag_disabled">NFC ist deaktiviert. Bitte aktivieren Sie es im Einstellungsmenü Ihres Gerätes.</string>
+  <string name="info_openhab_version_label">openHAB-Version</string>
+  <string name="info_openhab_apiversion_label">openHAB-Rest-API-Version</string>
+  <string name="info_openhab_uuid_label">openHAB-UUID</string>
+  <string name="info_openhab_secret_label">openHAB-Secret</string>
+  <string name="info_openhab_gcm_not_connected">Geräteregistrierung fehlgeschlagen</string>
+  <string name="info_openhab_gcm_connected">Gerät mit ID %s registriert</string>
+  <string name="action_settings">Einstellungen</string>
+  <string name="nfc_dialog_title">Schreibe eine Aktion für dieses Element in einen NFC Tag</string>
+  <string name="info_not_set">Nicht gesetzt</string>
+  <!-- Strings for MemorizingTrustManager activity -->
+  <string name="mtm_accept_cert">Unbekanntes Zertifikat akzeptieren?</string>
+  <string name="mtm_decision_always">Immer</string>
+  <string name="mtm_decision_once">Einmalig</string>
+  <string name="mtm_decision_abort">Abbrechen</string>
+  <string name="mtm_notification">Unbekanntes Zertifikat</string>
+  <string name="settings_openhab_none">Keins</string>
+  <!-- Themes -->
+  <string name="theme_name_light">Hell</string>
+  <string name="theme_name_dark">Dunkel</string>
+  <string name="theme_name_black">Schwarz/AMOLED</string>
+  <string name="theme_name_basic_ui">Basic UI</string>
+  <string name="theme_name_basic_ui_dark">Basic UI dunkel</string>
+  <!-- NFC toggle -->
+  <string name="nfc_action_on">An</string>
+  <string name="nfc_action_up">Hoch</string>
+  <string name="nfc_action_off">Aus</string>
+  <string name="nfc_action_down">Runter</string>
+  <string name="nfc_action_toggle">Umschalten</string>
+  <string name="nfc_action_current_color">Momentane Farbe</string>
+  <string name="nfc_action_to_sitemap_page">Zur entsprechenden Sitemap navigieren</string>
+  <!-- Drawer -->
+  <string name="drawer_open">Sitemap-Bereich öffnen</string>
+  <string name="drawer_close">Sitemap-Bereich schließen</string>
+  <string name="mainmenu_openhab_voice_recognition">Sprachbefehl</string>
+  <!-- About -->
+  <string name="about_title">Über</string>
+  <string name="about_license_title">Lizenz</string>
+  <string name="about_links">Links</string>
+  <string name="about_links_list"><![CDATA[\u00B7 <a href="https://github.com/openhab/openhab.android">openHAB-App-Quellcode</a><br><br> \u00B7 <a href="https://github.com/openhab/openhab.android/issues">Ein Problem berichten</a><br><br> \u00B7 <a href="http://docs.openhab.org/">openHAB-Dokumentation</a><br><br> \u00B7 <a href="https://community.openhab.org/">Community-Forum</a>]]></string>
```

Comment:
@mueller-ma is this going to be an issue?

Reply:
Tested it and works!